src/re_frame/middleware.cljs

```diff
@@ -189,13 +164,13 @@
       (fn after-handler
         [db v]
         (let [new-db (handler db v)]
-          (f new-db v)   ;; call f for side effects
+          (f new-db v)                                      ;; call f for side effects
```

Comment:
Sorry, I have reformatted it with Intellij/Cursive, it is indenting comment like that. 

Feel free to edit it. I won't be touching it that much.